internal/svcctl/svcctl.go

```diff
@@ -93,7 +101,7 @@ func StopServer(ipComm IPCommunicator) error {
 
 	err := stopAndWait(ctx, ipComm)
 	if err != nil && !errs.Matches(err, &ipc.ServerDownError{}) {
-		return errs.Wrap(err, "Cannot stop ipc server")
+		return err
```

Comment:
Not an improvement.

Reply:
The caller would report "Cannot stop server", then this function would report "Cannot stop and wait to confirm stop", then stopAndWait would report "Because {reasons}". So I found the message in this function redundant.